.env

```diff
@@ -13,4 +13,6 @@ API_STAC_ENDPOINT='https://staging.openveda.cloud/api/stac'
 GOOGLE_FORM = 'https://docs.google.com/forms/d/e/1FAIpQLSfGcd3FDsM3kQIOVKjzdPn4f88hX8RZ4Qef7qBsTtDqxjTSkg/viewform?embedded=true'
 
 FEATURE_NEW_EXPLORATION = 'TRUE'
-SHOW_CONFIGURABLE_COLOR_MAP = 'FALSE'
\ No newline at end of file
+COOKIE_CONSENT_FORM = 'TRUE'
```

Comment:
If this is something that can be tweaked through `veda.config`,  another env flag here seems to be redundant. What do you think?

Reply:
I believe so, I think there is some confusion on my end concerning the relationship between the `veda-ui` and `veda-config` codebases and the ultimate implementation of the feature flags. I could probably benefit from a quick connect to correct some assumptions on my end.